pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+      </li>
+      <li>
+        <strong>new:</strong> <code>$theme-table-stripe-text-color: default</code><br>
+        <strong>old:</strong> <code>"ink"</code>
+      </li>
+      <li>
+        <strong>new:</strong> <code>$theme-table-text-color: default</code><br>
+        <strong>old:</strong> <code>"ink"</code>
+      </li>
+    </ul>
+    <h6>What to do</h6>
+    <ol>
+      <li>Check your settings to see if they are set to the old default.</li>
+      <li>If they use the old default, delete the setting from your settings file so it uses the system default.</li>
+    </ol>
+  </div>
+  
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a5"
+    >
+      If you have code older than USWDS 2.10.1
+    </button>
+  </h4>
+  <div id="m-a5" class="usa-accordion__content usa-prose">
+    <p><strong><span class="usa-tag bg-accent-warm-darker">Settings</span> We updated some settings defaults:</strong></p>
+    <ul>
+      <li>
+        <strong>Updated: </strong>$theme-breadcrumb-background-color: default<br>
+        <strong>Was: </strong>white 
+      </li>
+      <li>
+        <strong>Updated: </strong>$theme-alert-icon-size: 5 <br>
+        <strong>Was: </strong>4
+      </li>
+    </ul>
+    <p><strong>What to do:</strong></p>
+    <ol>
+      <li>Search your codebase for any instances of <strong>the old theme</strong></li>
+      <li>Replace it with <strong>the new setting</strong></li>
+    </ol>
+    <p><strong><span class="usa-tag bg-accent-cool-darker">Markup</span> We updated usa-footer__logo-heading to use a p instead of an h3.</strong> We improved the accessibility of the footer by converting a non-semantic heading into paragraph text.</p>
+    <p><strong>What to do: </strong></p>
+    <ol>
+      <li>TK</li>
+      <li>TK</li>
+    </ol>
+  </div>
+</div>
+
+### 3. Install the USWDS 3.0 package
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+1. In a terminal window, navigate to your project root. 
+2. Install USWDS 3.0 with **npm install @uswds/uswds --save --save-exact**
+3. Uninstall USWDS 2 with **npm uninstall uswds**
+
+### 4. Update your Sass compiler settings and recompile CSS
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+The location of the USWDS source files is different in USWDS 3.0. You'll need to update your compiler settings to compile your Sass from USWDS 3.0.
+
+<div class="usa-accordion usa-accordion--bordered">
+  <!-- Use the accurate heading level to maintain the document outline -->
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a6"
+    >
+      If you're using USWDS Gulp
+    </button>
+  </h4>
+  <div id="m-a6" class="usa-accordion__content usa-prose">
+    <ol>
+      <li>Search for a line like <strong>const USWDS = "node_modules/uswds/dist"</strong> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
```

Comment:
The code snippet `node_modules/uswds/dist` is actually buried in a file inside `node_modules/uswds-gulp/config/uswds.js`, which makes it a bit harder to find in a search. The [gulpfile](https://github.com/uswds/uswds-gulp/blob/main/gulpfile.js#L22) that we ask people to copy over uses `const uswds = require("./node_modules/uswds-gulp/config/uswds");`. Could we use that instead? It has the bonus of using the term `uswds-gulp` in the search